### PR TITLE
Add support in collection list for custom title and image overrides

### PR DESCRIPTION
**Pull request by galenking**
### Why are these changes introduced?

1. The collection list section uses the title and image of the collection
2. This can be problematic when rebuilding a site using Dawn as existing collections may have images and titles that don’t fit the new design but making changes to the collection would impact the live theme
3. Often, merchants need to use different titles and/or images in collection lists than what is set at the collection level
4. Enables support for collection metafields for titles and/or images in collection lists

### What approach did you take?

1. Low-code/minimal changes with graceful fallbacks

### Visual impact on existing themes

1. There is no visual change on the frontend
2. Adds two block-level theme settings

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Comment by galenking** [comment]
My apologies. I am still new to creating PRs and my first attempt had some code from a different branch that should not have been included.

File: README.md [hunk 1]
```diff
@@ -39,12 +39,12 @@ git remote -v
 ```
 3. If you don't see an `upstream`, you can add one that points to Shopify's Dawn repository:
 ```sh
-git remote add upstream https://github.com/Shopify/dawn.git
+git remote add upstream https://github.com/lucidcommerceco/lucid-dawn.git
```
Comment (tyleralsbury) [inline README.md:42]: Will need to revert this change.

File: README.md [hunk 2]
```diff
@@ -39,12 +39,12 @@ git remote -v
...
 ```
 4. Pull in the latest Dawn changes into your repository:
 ```sh
 git fetch upstream
-git pull upstream main
+git pull upstream lucid-main
```
Comment (tyleralsbury) [inline README.md:47]: This one, too.

File: shopify.theme.toml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[environments.primary]
+store = ""
```
Comment (tyleralsbury) [inline shopify.theme.toml:2]: Will need to remove this file.

**Review by tyleralsbury — COMMENTED** [review]
Thanks for this contribution, Galen.

A few things need to be cleaned up that look like they were carried over from your fork of Dawn.

@melissaperreault What do you think about us including this as part of an upcoming release?  Seems like a nice feature add.

**Comment by galenking** [comment]
@tyleralsbury, I am so sorry! I totally messed this up when I deleted my repo 🤦🏼‍♂️ In any case, there was a bug in this PR with the image aspect ratio, which I have now fixed in the new PR (see https://github.com/Shopify/dawn/pull/3319). I hope this hasn’t screwed things up for you!
